package.json

```diff
@@ -61,35 +61,34 @@
     "@testing-library/jest-dom": "^5.16.2",
     "@testing-library/react": "^11.2.7",
     "@testing-library/react-hooks": "^5.1.1",
+    "@types/jest": "^27.4.1",
+    "@typescript-eslint/eslint-plugin": "^5.14.0",
+    "@typescript-eslint/parser": "^5.14.0",
     "babel-jest": "^26.6.3",
-    "eslint": "^7.24.0",
-    "eslint-config-prettier": "^8.2.0",
-    "eslint-plugin-jest-dom": "^3.8.0",
-    "eslint-plugin-react-hooks": "^4.2.0",
-    "eslint-plugin-testing-library": "^4.0.1",
+    "eslint": "^8.10.0",
+    "eslint-config-prettier": "^8.5.0",
+    "eslint-plugin-jest": "^26.1.1",
+    "eslint-plugin-jest-dom": "^4.0.1",
+    "eslint-plugin-react": "^7.29.3",
+    "eslint-plugin-react-hooks": "^4.3.0",
+    "eslint-plugin-testing-library": "^5.0.6",
     "fast-check": "^2.14.0",
-    "jest": "^26.6.3",
-    "prettier": "^2.2.1",
+    "jest": "^27.1.3",
+    "prettier": "^2.5.1",
     "prop-types": "^15.7.2",
     "react": "^16.14.0",
     "react-dom": "^17.0.2",
     "regenerator-runtime": "^0.13.9",
     "rimraf": "^3.0.2",
     "rollbar": "^2.21.1",
     "rollup": "^2.44.0",
-    "rollup-plugin-peer-deps-external": "^2.2.4"
+    "rollup-plugin-peer-deps-external": "^2.2.4",
+    "ts-jest": "^27.1.3",
+    "typescript": "^4.6.2"
   },
   "peerDependencies": {
     "react": "^16.0.0-0 || ^17.0.0-0",
     "prop-types": "^15.7.2",
     "rollbar": "^2.21.1"
-  },
-  "jest": {
-    "roots": [
-      "src/tests"
-    ],
-    "setupFilesAfterEnv": [
-      "./src/tests/jest-setup.js"
-    ]
```

Comment:
The jest config has been moved to its own file. This makes it easier for related configurations to reference it.